### PR TITLE
Export a minimal model

### PR DESCRIPTION
**Pull request by macumber**

### CODE REVIEW

File: lib/measures/check_model/resources/edge_properties.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   
   "type": "object",
   "properties": {
-    "edge_material": { "type": "string" },
-    "required": []
+    "length": { "type": "number" },
```
Comment (macumber) [inline lib/measures/check_model/resources/edge_properties.json:9]: @rd2 this schema applies only to the attributes of the Topolys Edge object

File: lib/measures/check_model/resources/face_properties.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
   
   "type": "object",
   "properties": {
-    "psi_value": { "type": "number" },
-    "required": ["psi_value"]
+    "surface_name": { "type": "string" },
```
Comment (macumber) [inline lib/measures/check_model/resources/face_properties.json:9]: @rd2 this schema applies only to the attributes of the Topolys Face object

File: lib/topolys_measures/schema/minimal_export.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://json-schema.org/automaticmagic/topolys.json#",
+  "title": "Topolys Measures Edge Properties Schema",
+  "description": "Schema for the Edge Properties for Topolys Measures",
+  
+  "type": "object",
```
Comment (macumber) [inline lib/topolys_measures/schema/minimal_export.json:7]: @rd2 if we want a custom export we can create a new schema

File: lib/measures/check_model/measure.rb [hunk 4]
```diff
@@ -122,7 +143,27 @@ def run(model, runner, user_arguments)
     # install graphviz and make sure dot is in the system path
     tpm.save_graphviz('shell.dot')
     system('dot shell.dot -Tpdf -o shell.pdf')
-
+    
+    # save the full model
+    tpm.save('full_tpm.json')
+    
+    # create a minimal model
```
Comment (macumber) [inline lib/measures/check_model/measure.rb:150]: @rd2 here we just transform the Topolys objects to export the schema we want

Comment (rd2) [reply lib/measures/check_model/measure.rb:150]: @macumber thanks for the example code.

File: lib/topolys_measures/schema/minimal_export.json (hunk 3 above)
Comment (rd2) [reply lib/topolys_measures/schema/minimal_export.json:7]: @macumber holding specific schemas for distinct tasks (e.g. topolys.json for Topolys arborescence and internal objects; tbd.json for TBD wrapper, and so on) always made more sense to me (that being said, I suspect each schema's "id" would need to be set accordingly)

Comment (macumber) [reply lib/topolys_measures/schema/minimal_export.json:7]: Yes you are right!

**Pull request merged**